Demo/Demo/SwiftUIComponents/CardVaultViews/CardVaultView.swift

```diff
@@ -5,30 +5,36 @@ struct CardVaultView: View {
     @StateObject var baseViewModel = BaseViewModel()
     @StateObject var cardVaultViewModel = CardVaultViewModel()
 
+    @State var intent: String = "AUTHORIZE"
```

Comment:
I think I will fix this in the SwiftUI conversion PR because I want to use the exiting enum Intent that is in its separate file but is used in BaseViewModel which I delete in that PR.

Reply:
Ok - maybe let's leave a TODO note to use that enum then